<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor</class>
 <widget class="QMainWindow" name="TextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <family>UbuntuMono Nerd Font Propo</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Type your text here...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="searchReplaceWidget" native="true">
      <layout class="QHBoxLayout" name="searchReplaceLayout">
       <item>
        <widget class="QLineEdit" name="searchInput">
         <property name="placeholderText">
          <string>Search...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="replaceInput">
         <property name="placeholderText">
          <string>Replace...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="replaceButton">
         <property name="text">
          <string>Replace</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionEncrypt"/>
    <addaction name="actionDecrypt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionEncrypt">
   <property name="text">
    <string>Encrypt</string>
   </property>
  </action>
  <action name="actionDecrypt">
   <property name="text">
    <string>Decrypt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
